generators/resources/mappings.v1.json.template

```diff
@@ -3,6 +3,7 @@
     {{#each credentialKeys}}
     "{{../serviceKey}}_{{this}}" : {
         "searchPatterns": [
+            "user-provided:{{../serviceName}}:{{access ../map this ../nestedJSON}}",
```

Comment:
@quanvo87 Shouldn't there be an underscore between the serviceName and the key like this

```
user-provided:{{../serviceName}}_{{access ../map this ../nestedJSON}}